### PR TITLE
Allow Use of Curves and Tables for Wind Pressure Coefficient Profiles

### PR DESCRIPTION
**Pull request by jasondegraw**
Pull request overview
---------------------
Allow use of regular curves and tables for specifying wind pressure coefficients for use in AirflowNetwork models. The current input objects are still supported, but are now read in as tables with one independent variable. Additional changes include support for using relative wind angles (relative to the surface normal) and symmetric curves (only 180 degrees need to be specified). In terms of inputs, there are two additional fields added to the external node object and one field removed from the AFN simulation control object. There are likely going to be some small diffs, as the table interpolation is not handled in exactly the same way. 

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
   - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
   - NewFeature: This pull request includes code to add a new feature to EnergyPlus
   - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
   - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Code style (parentheses padding, variable names)
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check -- verify this
 - [ ] Unit Test(s)
 - C++ checks:
   - [ ] Argument types
 - IDD changes:
   - [ ] Verify naming conventions and styles, memos and notes and defaults
   - [ ] Open windows IDF Editor with modified IDD to check for errors
   - [ ] If transition, add rules to spreadsheet
   - [ ] If transition, add transition source
   - [ ] If transition, update idfs
 - [ ] If new idf included, locally check the err file and other outputs
 - [ ] Documentation changes in place
 - [ ] Changed docs build successfully
 - [ ] If output changes, including tabular output structure, add to output rules file for interfaces

### CODE REVIEW

**Comment by jasondegraw** [comment]
@Myoldmopar @mjwitte I'd like to see additional CI reports (who knows when that will happen) but I believe all of the diffs are expected.

File: idd/Energy+.idd.in [hunk 1]
```diff
@@ -23374,11 +23370,30 @@ AirflowNetwork:MultiZone:ExternalNode,
       \units m
       \default 0.0
       \note Designates the reference height used to calculate relative pressure.
- A2 ; \field Wind Pressure Coefficient Values Object Name
+ A2 , \field Wind Pressure Coefficient Curve Name
       \required-field
       \type object-list
       \object-list WPCValueNames
-      \note Enter the name of the AirflowNetwork:MultiZone:WindPressureCoefficientValues object.
```
Comment (mjwitte) [inline idd/Energy+.idd.in:23381]: Is a WPCValueName still valid here?  If not, then delete the object-list WPCValueNames.  If so, then keep this note plus the new note.

File: idd/Energy+.idd.in [hunk 2]
```diff
@@ -23374,11 +23370,30 @@ AirflowNetwork:MultiZone:ExternalNode,
...
+      \note The name of the curve or table object specifying the wind pressure coefficient.
+ A3 , \field Symmetric Wind Pressure Coefficient Curve
+      \type choice
+      \key Yes
+      \key No
+      \default No
+      \note Specify whether the pressure curve is symmetric or not.
+      \note Specify Yes for curves that should be evaluated from 0 to 180 degrees
+      \note Specify No for curves that should be evaluated from 0 to 360 degrees
+ A4 ; \field Use Relative Wind Angle
```
Comment (mjwitte) [inline idd/Energy+.idd.in]: If I remember correctly, you were going to change this field name and keys?

**Review by mjwitte — COMMENTED** [review]
Reviewed idd changes only.  Just a couple of comments.

File: idd/Energy+.idd.in (hunk 1 above)
Comment (jasondegraw) [reply idd/Energy+.idd.in:23381]: Yes, it is. If it is OK, I'll combine the old and new notes into one note.

File: idd/Energy+.idd.in (hunk 2 above)
Comment (jasondegraw) [reply idd/Energy+.idd.in]: That's correct. I changed the code but not the IDD. Will fix momentarily.

File: idd/Energy+.idd.in (hunk 1 above)
Comment (mjwitte) [reply idd/Energy+.idd.in:23381]: Yep, that would be good.

**Comment by Myoldmopar** [comment]
OK, this is ready.  Final thoughts before merge....5 minutes and counting.

**Pull request merged**